waspc/src/Wasp/Generator/Job/Process.hs

```diff
@@ -167,6 +167,6 @@ makeNodeVersionMismatchMessage nodeVersion =
 waspNodeRequirementMessage :: String
 waspNodeRequirementMessage =
   unwords
-    [ "Wasp requires node " ++ show C.nodeVersionBounds ++ ".",
+    [ "Wasp requires Node.js to be the latest LTS version, which is currently v" ++ show (SV.major C.nodeVersion) ++ ".",
```

Comment:
Keep in mind that such wording calls for a quick update as soon as Node releases a new LTS version, which might be a good thing.

Still, what if we don't want to support the newest LTS right away? We can remove the line, but all existing installations of Wasp will technically be giving out wrong information.

For example, Node 18 becomes the new LTS version. We don't want to update it because it breaks something. Even if we change the wording here, all existing Wasp installations still say "Wasp requires Node.js to be the latest LTS version, which is currently v16.". This now becomes incorrect and can potentially give an impression that the project is out of date or poorly maintained.

Reply:
Also, I see you've swapped `^16.0.0` with `v16` probably because the first variant, although technically more informative, requires the users to understand semver syntax.

Still, saying just `v16` might leave them with the impression that their version must be 16.0.0, which is incorrect.

Perhaps the best option is to print something like `v16.x.x`. We can do this by changing `VersionBound`'s show method.

Reply:
Makes sense -> I will revert message to be what it was before, that is the easiest and most correct!